tests/main/preseed-snapd-snap/task.yaml

```diff
@@ -0,0 +1,105 @@
+summary: Check that preseeding of current ubuntu cloud image works with snapd
+  and core18 works.
+description: |
+  This test checks that preseeding of Ubuntu cloud images with snap-preseed
+  command works, up to the point where the image is ready to be booted.
+  The test assumes cloud image with a core and lxd snaps in its seeds/.
+
+systems: [ubuntu-20.04-*]
```

Comment:
Forgive my ignorance, why is this limited to Ubuntu 20.04?

Reply:
Because no ubuntu cloud image seeds core18+snapd yet, and 20.04 is the only possible candidate (or maybe this is even too optimistic assumption).

Reply:
Could you add this as a comment please.

Reply:
actually I think they might consider that switch for bionic too, because otherwise they will end up with core + core18 on those at some point

Reply:
Okay, I can enable it for more systems; if they start seeding snapd on 20.04 but not on others I can make setup_preseeding_core18 conditional and modify the image only where it is still needed.

Reply:
Enabled for 19.10 as well, but not for 18.04 since it doesn't do seeding at all at the moment. I will enable once it actually introduces seeds.